src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDependency.cs

```diff
@@ -631,6 +652,7 @@ private static void GetSerializedObject(ObjRef objRef, BinaryFormatter formatter
         [SecurityPermission(SecurityAction.Assert, Flags = SecurityPermissionFlag.SerializationFormatter)]
         private static SqlDependencyProcessDispatcher GetDeserializedObject(BinaryFormatter formatter, MemoryStream stream)
         {
+            formatter.Binder = new SqlDependencyProcessDispatcherSerializationBinder();
```

Comment:
Roslyn warning: `warning CA2301: The method 'object BinaryFormatter.Deserialize(Stream serializationStream)' is insecure when deserializing untrusted data without a SerializationBinder to restrict the type of objects in the deserialized object graph.`
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2301?view=vs-2019